docs/src/content/pages/fields/image.mdoc

```diff
@@ -1,11 +1,15 @@
 ---
 title: Image field
 ---
-The `image` field is used to upload an image.
+The `image` field is used to store an image. In the Admin UI it renders a button to choose an image file. When a file has been selected it shows a preview along with a button to remove.
```

Comment:
Sorry I missed this last review, but this second sentence may be too detailed? The Image field UI is something that I can foresee changing (e.g. to make a nice dropzone) so the description has a higher chance of getting out of date than other simpler fields. 

Perhaps simplify it? If we get the fields integrated the user will see what it looks like anyways.

```suggestion
The `image` field is used to store an image. In the Admin UI it renders an image picker component.
```